src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/MethodLengthCheck.java

```diff
@@ -126,16 +153,19 @@ private int getLengthOfBlock(DetailAST openingBrace, DetailAST closingBrace) {
     }
 
     /**
-     * Sets maximum length of a method.
-     * @param length the maximum length of a method.
+     * Setter to specify the maximum number of lines allowed.
+     *
+     *  @param length the maximum length of a method.
```

Comment:
@pbludov ping
double space again

Reply:
Done. I'll try to figure out why the extra space appears.